docs/install-docker-dev.md

```diff
@@ -1,3 +1,75 @@
 # Using Docker (experimental)
```

Comment:
I would leave the file in as I'm going to expand it later on when the Docker dev environment has got its revamp/improvements.